lib/pbio/drv/bluetooth/bluetooth_stm32_cc2640.c

```diff
@@ -2277,6 +2271,14 @@ PROCESS_THREAD(pbdrv_bluetooth_spi_process, ev, data) {
 
     PROCESS_BEGIN();
 
+    #if DEBUG_ON_LAST_UART_PORT
+    // Wait for the UART to be ready for debugging.
+    while (pbdrv_ioport_debug_uart_init(uart_poll_callback) != PBIO_SUCCESS) {
+        etimer_set(&timer, 100);
+        PROCESS_WAIT_EVENT_UNTIL(ev == PROCESS_EVENT_TIMER && etimer_expired(&timer));
```

Comment:
Why delay boot for 100 ms?

It should be fine to just yield here and skip the timer.